### PR TITLE
Remove "Big Wheel Brigade"

### LINKED ISSUE #1455: Big wheel brigade have ceased there operations in 2017.
We can remove them from the list as they are no longer operational.

### PR DESCRIPTION
**Pull request by xstable**
Close #1455 

My first Hacktoberfest PR 🙂 

Signed-off-by: Samuel Suther <info@suther.de>

📝Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]`

ℹ️Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

- [ ] This PR contains housekeeping only (URL edits, copy changes etc) - if check, delete other lines that don't apply.
- [ ] I am an employee of the company mentioned and confirm all included details are correct.
- [ ] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [ ] The company directly hires employees. No bootcamps / freelance sites / etc
- [ ] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [ ] A [company profile](https://github.com/remoteintech/remote-jobs/blob/main/company-profiles/example.md) is included - This is __required__ for new additions.
- [ ] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [ ] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [ ] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/tree/main/.github/CONTRIBUTING.md).

This Pull Request should adhere to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/tree/main/.github/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,7 +84,6 @@ Name | Website | Region
 [BetterUp](/company-profiles/betterup.md) | https://www.betterup.com/ | USA
 [BeyondPricing](/company-profiles/beyondpricing.md) | https://beyondpricing.com | USA, Europe
 [Big Cartel](/company-profiles/big-cartel.md) | https://www.bigcartel.com | USA
-[Big Wheel Brigade](/company-profiles/big-wheel-brigade.md) | https://www.bigwheelbrigade.com/ | USA
```
Comment (Christian-Oleson) [inline README.md:87]: Please also remove the company profile in the company-profiles folder.

**Comment by dougaitken** [comment]
Hi @xstable congrats on contributing to HF. 

As @Christian-Oleson mentioned, please delete the company profile markdown file, commit that change to your branch then push that to update the Pull Request.

File: README.md (hunk 1 above)
Comment (xstable) [reply README.md:87]: sure, one moment.

Comment (Christian-Oleson) [reply README.md:87]: Thank you!

**Pull request merged**